data/free worlds start.txt

```diff
@@ -1399,6 +1399,119 @@ mission "FW Pirates 3.1B"
+				`	It also looks as if they've brought some friends along. You notice a Republic carrier emitting strange energy signatures and launching fighters that are clearly alien in design.`
+				accept
+	on accept
+		set "fw pirates: own way"
+
+	npc kill
+		personality plunders staying
+		government "Pirate"
+		system Shaula
+		fleet "Large Southern Pirates"
+		fleet "Large Southern Pirates"
+		fleet "Large Southern Pirates"
+		fleet "Large Core Pirates"
+		fleet "Large Core Pirates"
+		fleet "Large Core Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Southern Pirates"
+		fleet "Large Southern Pirates"
+		fleet "Large Southern Pirates"
+		fleet "Large Core Pirates"
+		fleet "Large Core Pirates"
+		fleet "Large Core Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Marauder fleet I"
+		fleet "Marauder fleet II"
+		fleet "Marauder fleet III"
+		fleet "Marauder fleet IV"
+		fleet "Marauder fleet V"
+		fleet "Marauder fleet VI"
+		fleet "Marauder fleet VII"
+		fleet "Marauder fleet VIII"
+		fleet "Marauder fleet IX"
+		fleet "Marauder fleet X"
+		fleet "Marauder fleet I"
+		fleet "Marauder fleet II"
+		fleet "Marauder fleet III"
+		fleet "Marauder fleet IV"
+		fleet "Marauder fleet V"
+		fleet "Marauder fleet VI"
+		fleet "Marauder fleet VII"
+		fleet "Marauder fleet VIII"
+		fleet "Marauder fleet IX"
+		fleet "Marauder fleet X"
+		ship "Bastion (Heavy)" "Dread"
+		ship "Carrier (Alpha)" "Trafficker"
+		ship "Marauder Falcon (Engines)"
+		ship "Marauder Falcon (Engines)"
+		ship "Marauder Falcon (Engines)"
+		ship "Marauder Falcon (Engines)"
+		ship "Marauder Falcon (Engines)"
+		ship "Marauder Falcon (Engines)"
+		ship "Marauder Falcon (Engines)"
+		dialog "Songs will be written about your victory here today, but all you will remember is the screams of terrified, dying teenage pirate crews echoing over the radio, wave upon wave of incoming missiles, and the clouds of wreckage littering the system at the end. You shake yourself out of your reverie. You have work to do now on <destination>. It's time to finish what you've begun."
+	npc
+		government "Pirate"
+		system "Shaula"
+		personality heroic nemesis staying vindictive
+		ship "Korath Chaser" "Brutus"
+		ship "Korath Chaser" "Judas"
+		ship "Korath Chaser" "Benedict Arnold"
+		ship "Korath Chaser" "Alcibiades"
+		ship "Korath Chaser" "Guy Fawkes"
+		ship "Korath Chaser" "Ephialtes"
+		ship "Korath Chaser" "Quisling"
+		ship "Korath Chaser" "Tokyo Rose"
+		ship "Korath Chaser" "Doña Marina"
+		ship "Korath Chaser" "Mir Jaffar"
```

Comment:
The other time an Alpha carrier shows up in the game, it comes with 10 Kor Sestor drone fighters. I thought that the carrier should have some unusual fighters to go with it, and the Korath Chasers seemed more appropriate/available for this region of space.